server/errors.py

```diff
@@ -7,5 +8,4 @@ class RoomNotFoundError(Exception):
     def __init__(self, message: str) -> None:
         super().__init__(message)
 
-        self.code = StatusCode.ROOM_NOT_FOUND
-        self.message = message
+        self.data = Event(type=EventType.ERROR, data=ErrorData(message=message), status_code=StatusCode.ROOM_NOT_FOUND)
```

Comment:
`data=ErrorData(message=message).dict()` or pass in the data as a dictionary with the message key. Perhaps it would be easier to make do:
```py
self.message = {"message": message}
self.data = Event(type=EventType.ERROR, data=message, status_code=StatusCode.ROOM_NOT_FOUND)
```

The data in Event must be a mapping to use the `EventData(**value)` in events.py

Reply:
Fixed in 83f16e4997d25b12d4f4c162a66d293ae8f6ed6a.